budget/tests.py

```diff
@@ -758,7 +759,7 @@ def api_add_member(self, project, name, weight=1):
     def get_auth(self, username, password=None):
         password = password or username
         base64string = base64.encodestring(
-                '%s:%s' % (username, password)).replace('\n', '')
+            ('%s:%s' % (username, password)).encode()).decode().replace('\n', '')
```

Comment:
I think you saw encode().decode() and not encode()).decode(), no ? because for me there is no other way of doing this, the encode is necessary because base64.encodestring excepts bytes and the decode is necessary because bytes are returned and we want to use replace.